src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -51,7 +51,20 @@
     <SupplementalPayloadFile>$(ArchivesRoot)$(SupplementalPayloadFilename)</SupplementalPayloadFile>
     <OverwriteOnUpload Condition="'$(OverwriteOnUpload)' == ''">false</OverwriteOnUpload>
     <TimeoutInSeconds Condition="'$(TimeoutInSeconds)' == ''">600</TimeoutInSeconds>
-    <HelixApiEndpoint>https://helixview.azurewebsites.net/api/jobs</HelixApiEndpoint>
+    
+    <IsOfficial Condition="'$(OfficialBuildId)'!=''">true</IsOfficial>
+    <CurrentDate>$([System.DateTime]::Now.ToString(yyyyMMdd.mmss))</CurrentDate>
+    <HelixJobType Condition="'$(FilterToTestTFM)'=='netcore50' and !$(TestNugetRuntimeId.Contains('aot'))">test/functional/uwp/</HelixJobType>
+    <HelixJobType Condition="'$(FilterToTestTFM)'=='netcore50' and $(TestNugetRuntimeId.Contains('aot'))">test/functional/ilc/</HelixJobType>
+    <HelixJobType Condition="$(FilterToTestTFM.Contains('net46'))">test/functional/desktop/cli/</HelixJobType>
+    <HelixJobType Condition="'$(HelixJobType)'=='' And '$(Performance)'=='true'">test/functional/perf/cli/</HelixJobType>
+    <HelixJobType Condition="'$(HelixJobType)'==''">test/functional/cli/</HelixJobType>    
```

Comment:
This all looks very tightly coupled, but weakly regulated.  ie, it would be easy for someone to miss updating one of these values.   A structure like this would be slightly more flexible / manageable...

```xml
  <PropertyGroup>
    <HelixJobTypeBase Condition="'$(HelixJobTypeBase)' == ''">test/functional/</HelixJobTypeBase>
  </PropertyGroup>
  <PropertyGroup>
    <HelixJobTypeModifier Condition="'$(FilterToTestTFM)'=='netcore50' and !$(TestNugetRuntimeId.Contains('aot'))">uwp/</HelixJobType>  
    <HelixJobTypeModifier Condition="'$(FilterToTestTFM)'=='netcore50' and $(TestNugetRuntimeId.Contains('aot'))">ilc</HelixJobTypeModifier >  
    <HelixJobTypeModifier Condition="$(FilterToTestTFM.Contains('net46'))">desktop/cli</HelixJobTypeModifier >  
    <HelixJobTypeModifier Condition="'$(HelixJobTypeModifier )'=='' And '$(Performance)'=='true'">perf/cli</HelixJobTypeModifier >  
    <HelixJobTypeModifier Condition="'$(HelixJobTypeModifier )'==''">cli</HelixJobTypeModifier >      
  </PropertyGroup>
  <PropertyGroup>
    <HelixJobType Condition="'$(HelixJobType)' == ''">$(HelixJobTypeBase)$(HelixJobTypeModifier)/</HelixJobType>
  </PropertyGroup>
```